### DISCUSSION TITLE
How to hide the green highlighted menu of the map markers?

### DISCUSSION THREAD

**Discussion by WenRob967** [osmandapp/OsmAnd#19631] [Q&A]
How can I hide the green highlighted menu of the map markers from the normal map view?
 
 
 
![OSMAND_Screen-Image](https://github.com/osmandapp/OsmAnd/assets/167696970/a2eaa285-2ec5-49f3-9b16-c9863a127adf)

**Comment by dmpr0** [comment]
Hi! This is widget in the Top panel. In order to edit widget list, you need to open 
* **Menu** – **Configure Screen** (make sure that needed profile is selected)
* Tap on **Top panel**, edit
* Find **Map markers bar**, remove it.
